overrides/config/DimensionalEdibles.cfg

```diff
@@ -314,6 +317,42 @@ dimensionaledibles {
             }
 
         }
+        
+        ##########################################################################################################
+        # islandcake
+        #--------------------------------------------------------------------------------------------------------#
+        # The category for dealing with the Island Cake
+        ##########################################################################################################
+
+        islandcake {
+            # Allow players to create personal islands when on a FTB Team
+            B:allowPersonalIslands=true
+
+            # Set to true to make the Island Cake consume fuel.
+            B:consumesFuel=true
+
+            # Set the fuel used by Island Cake (Don't change this unless you know what you're doing).
+            S:fuel=minecraft:cobblestone
```

Comment:
Can this be changed to pulsating dust?

Reply:
Yep, just change it to the internal name